Yubico.YubiKey/src/Yubico/YubiKey/YubiKeyFeature.cs

```diff
@@ -15,251 +15,256 @@
 namespace Yubico.YubiKey
 {
     /// <summary>
-    /// Named YubiKey features or distinct behaviors that have been added or changed over the years that can be queried.
+    ///     Named YubiKey features or distinct behaviors that have been added or changed over the years that can be queried.
```

Comment:
is this the correct format (in the whole file)?

Reply:
Yes, its an automatic formatting feature in Rider that properly formats not only code, but documentation as well